### ISSUE TITLE
Plugin does not support EndpointRouteBuilder interface

### ISSUE THREAD

**Issue by DevCam** [camel-tooling/camel-idea-plugin#941]
When using a `EndpointRouteBuilder` as follows:

```
public class Example extends EndpointRouteBuilder {

  @Override
  public void configure() throws Exception {

    from(direct("foo"))
            .log("hello!");

    from(timer("bar"))
            .to(direct("foo"));

    from(timer("baz"))
            .to("direct:foo");
  }
}

```

- The plugin cannot navigate to the route definition (on `bar`)
- The plugin reports an error because the route definition is not found ( on `baz)

**Comment by davsclaus** [comment]
Yeah something is a bit wrong, this has 4 camel icons in a row

<img width="1531" alt="Screenshot 2023-10-13 at 19 40 22" src="https://github.com/camel-tooling/camel-idea-plugin/assets/477100/6f793eb0-6d1c-4796-8b01-db1989953cc1">

**Comment by davsclaus** [comment]
@ammachado did your recent PR fix this ?

**Comment by ammachado** [comment]
@davsclaus No, but I got a failing test case for it.

**Comment by davsclaus** [comment]
@rastislavpapp not sure if we keep seeing those repeat camel icons with latest code and when using Endpoint DSL.

**Comment by rastislavpapp** [comment]
no, this was fixed in #1136
